lntest/rpc/watchtower.go

```diff
@@ -52,6 +52,38 @@ func (h *HarnessRPC) AddTower(
 	return resp
 }
 
+// DeactivateTower makes an RPC call to the WatchtowerClient of the given node
+// and asserts.
+func (h *HarnessRPC) DeactivateTower(req *wtclientrpc.DeactivateTowerRequest) {
+	ctxt, cancel := context.WithTimeout(h.runCtx, DefaultTimeout)
+	defer cancel()
+
+	_, err := h.WatchtowerClient.DeactivateTower(ctxt, req)
+	h.NoError(err, "DeactivateTower")
+}
+
+// TerminateSession makes an RPC call to the WatchtowerClient of the given node
+// and asserts.
+func (h *HarnessRPC) TerminateSession(
+	req *wtclientrpc.TerminateSessionRequest) {
+
+	ctxt, cancel := context.WithTimeout(h.runCtx, DefaultTimeout)
+	defer cancel()
+
+	_, err := h.WatchtowerClient.TerminateSession(ctxt, req)
+	h.NoError(err, "TerminateSession")
+}
+
+// RemoveTower makes an RPC call to the WatchtowerClient of the given node
+// and asserts.
+func (h *HarnessRPC) RemoveTower(req *wtclientrpc.RemoveTowerRequest) {
```

Comment:
ah yeah forgot to call this. Added basic calls to this at the very end of the test 👍 

>  It would be nice to demonstrate in which cases a watchtower can and cannot be removed

towers should always be removable now that un-acked updates are replayed onto the backup queue. 
This is all tested in detail in a couple of the test cases in `wtclient/client_test.go` where it is easier to contrive the various situations :) especially the following tests:
- "switch to new tower after tasks are bound"
- "can remove due to un-acked updates (no client restart)"
- "can remove tower with un-acked updates (with restart)"